apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx

```diff
@@ -148,10 +189,68 @@ const RemoteSvgTest: React.FunctionComponent = () => {
   );
 };
 
+const CustomSvgInputTest: React.FunctionComponent = () => {
+  const defaultxml = `
+  <svg width="256" height="256" viewBox="0 0 48 48">
+  <rect width="48" height="48" fill="red"/>
+  </svg>
+`;
+
+  const textInputStyle = { height: 256, width: 1024, borderColor: 'gray', borderWidth: 1, fontSize: 16 };
+  const [inputText, setInputText] = useState<string>(defaultxml);
+  const [svgText, setSvgText] = useState<string>(defaultxml);
+  const [dataText, setDataText] = useState<string>('click draw to see parsed data');
+  const [showParse, setShowParsed] = useState<boolean>(false);
+
+  const handleChangeText = React.useCallback(
+    (text: string) => {
+      setInputText(text);
+    },
+    [setInputText],
+  );
+
+  return (
+    <View>
+      <Text>
+        Paste an svg in here and click Draw to try it out. The way it uses rnsvg&apos;s parse function to attempt to validate svg may cause
+        asserts and crashes if the svg is invalid. Looking for another solution for this, but this is still useful in its current state.
+      </Text>
+      <TextInput
+        multiline={true}
+        accessibilityLabel="Custom svg text input"
+        value={inputText}
+        style={textInputStyle}
+        onChangeText={handleChangeText}
+      />
+      <View style={{ flexDirection: 'row' }}>
+        <Button
+          onClick={() => {
```

Comment:
useCallback for the onClick callback here too, please.

Same with Toggle button below